stdlib/public/Python/Python.swift

```diff
@@ -1071,14 +1097,16 @@ extension Range : PythonConvertible where Bound : PythonConvertible {
     guard pythonObject.step == Python.None else { return nil }
     self.init(uncheckedBounds: (lowerBound, upperBound))
   }
+}
 
+extension PartialRangeFrom : PythonConvertible where Bound : PythonConvertible {
   public var pythonObject: PythonObject {
     _ = Python // Ensure Python is initialized.
-    return Python.slice(lowerBound, upperBound, Python.None)
+    return Python.slice(lowerBound, Python.None, Python.None)
   }
 }
 
-extension PartialRangeFrom : PythonConvertible where Bound : PythonConvertible {
+extension PartialRangeFrom : ConvertibleFromPython where Bound : ConvertibleFromPython {
```

Comment:
Fit within 80 columns.
```suggestion
extension PartialRangeFrom : ConvertibleFromPython
  where Bound : ConvertibleFromPython {
```